modules/graphdb/user_data.tf

```diff
@@ -41,8 +41,8 @@ data "cloudinit_config" "graphdb_user_data" {
       ebs_volume_size : var.ebs_volume_size
       ebs_volume_iops : var.ebs_volume_iops
       ebs_volume_throughput : var.ebs_volume_throughput
-      ebs_kms_key_arn : var.ebs_kms_key_arn
       device_name : var.device_name
+      graphdb_ebs_kms_key_arn : var.graphdb_ebs_key_arn
```

Comment:
Do we have another set of ebs ? just name it ebs_kms_key_arn and group it with the rest..